background/storage/local.ts

```diff
@@ -69,11 +69,29 @@ export async function getStorage() {
       { override: true }
     )
     storage.subscribe(async (state) => {
-      console.log("[background] Sync state to popup")
-      await browser.runtime.sendMessage(browser.runtime.id, {
-        action: StorageAction.Sync,
-        state
-      })
+      console.log(
+        "[background] Sync state to app/storage/index.ts (in tabs/index.html)"
```

Comment:
I don't think this message is more informative. It is too specific to file structure.